modules/assim.sequential/R/Remote_helpers.R

```diff
@@ -235,29 +256,47 @@ SDA_remote_launcher <-function(settingPath,
                            scratchdir = ".")
   }
   
-
+  # Do the Rsync to copy all the main dir of the inputs
+  need.copy <- input.paths[!missing.inputs]
+  
+  need.copy.dirs <- dirname(need.copy) %>%
+    unique() %>%
+    discard(~ .x %in% c(".", "/fs/data1/pecan.data/dbfiles" ))
```

Comment:
You can't hard code paths like this

Reply:
willing to accept this path for now since it it seems to just be filtering some stuff out, but still agree it's not a great long-term solution